database/migrations/2019_10_31_060447_create_questions_table.php

```diff
@@ -14,10 +14,8 @@ class CreateQuestionsTable extends Migration
     public function up()
     {
         Schema::create('questions', function (Blueprint $table) {
-            $table->Integer('id')->autoIncrement();
```

Comment:
please call method from object with lowercase, use `integer` instead of `Integer` as this syntax is now depreciated.